src/components/data-management/AddMetadataButton.jsx

```diff
@@ -27,7 +30,7 @@ const AddMetadataButton = ({ samplesTableRef }) => {
   const isSubsetted = activeExperiment?.isSubsetted;
   const samples = useSelector((state) => state.samples);
   const selectedTech = samples[activeExperiment?.sampleIds[0]]?.type;
-
```

Comment:
this `components/data-management` folder is becoming huge. Consider creating a new folder for cell level metadata related components. In the future, we can also create folders for the other components. In terms of naming convention -- not sure here. data-exploration and data-processing seem to be following completely different conventions

Reply:
good suggestion, i can see how we can create folders : project, metadata, experiment, fileUpload for the categories